platform-enterprise_versioned_docs/version-25.1/tutorials/retry-strategy.md

```diff
@@ -0,0 +1,79 @@
+
+This guide outlines best practices for mitigating the impact of Spot interruptions and ensuring critical tasks can retry or recover reliably.
+
+## Recommended mitigations
+
+### Use an On-Demand compute environment
+
+For workflows with a significant proportion of long-running processes, the costs of, and mitigations necessary for working with Spot may outweigh the benefits. You may find it simpler and even, possibly, cheaper to simply run those workloads in On-Demand compute environments.
+
+### Move long-running tasks to On-Demand 
+
+Tasks with long runtimes are particularly vulnerable to Spot termination. In Platform, you can explicitly assign critical or long-duration tasks to On-Demand queues and leave other tasks to run in a default Spot queue by default:
+
+```bash
+process {
+	withName: 'run_bcl2fastq' {
+	     queue = 'TowerForge-MyOnDemandQueue'
+	} 
+}
+```
+
+If you don’t already have one, you may need to create an On-Demand compute environment in the Seqera Platform. Once it’s available, you can find the corresponding On-Demand queue name by navigating to **Compute Environments** in the Platform UI. Locate the configuration for your specific On-Demand environment, then scroll down to the **Manual Config Attributes** section. This section lists key configuration details, including queue names. Look for the queue name prefixed with `TowerForge-` if it was created by Forge.
+
+### Use retry strategies for Spot Interruptions
+
+#### Handle retries in Nextflow by setting `errorStrategy` and `maxRetries`
+
+A simple generic retry strategy at the Nextflow level can be more appropriate where run times are sufficiently low that retries are likely to succeed. This can be configured as follows:
+
+```bash
+process {
+   errorStrategy = 'retry'
+   maxRetries = 3 
+}
+```
+
+This example configuration will apply to all types of job failure. Because Spot reclamations do not produce diagnostic exit codes, it is currently not possible to configure retries at the Nextflow level specifically for reclamations. Note that, given the escalating costs of repeated retries, an On-Demand queue is likely a more cost-effective option than very large numbers of retries. If you still see failures after applying configuration like this, solutions involving On-Demand queues are likely to be more effective at limiting costs and runtimes.
+
+#### Handle retries in AWS by setting `aws.batch.maxSpotAttempts`
+
+If you do not have processes with long runtimes in your workflow (i.e., all processes are under 1 hour runtime), you can consider configuring automatic retries in case of interruption:
```

Comment:
```suggestion
If all processes in your workflow have runtimes short enough to feasibly complete before reclamation, you can consider configuring automatic retries in case of interruption:
```